scripts/semaphore_method.py

```diff
@@ -0,0 +1,128 @@
+# coding: utf-8
```

Comment:
Rename this to `semaphore_utils` instead.

Reply:
Done.